buddychatcli/Person.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+
+using CommandLine;
+
+namespace BuddyChatCLI
+{
+
+    public class Person
+    {
+        
+        public string username { get; set; }
+
+        public string name { get; set; }
+
+        public string pronoun { get; set; }
+
+        public bool audioChatOnly { get; set; }

```

Comment:
Should we ignore this - I feel that in the end we stopped doing anything with this data